src/res/translation/translation_nl_NL.ts

```diff
@@ -417,7 +417,7 @@
     <message>
         <location filename="../../audiomixerboard.cpp" line="229"/>
         <source>PAN</source>
-        <translation>PAN</translation>
+        <translation>BAL.</translation>
```

Comment:
Are you sure about the dot here?

Reply:
This is the abbreviated form, hence the dot. Full form is "BALANS". For some reason there is also abbreviation in the English (GRP), probably to avoid having long labels, so followed example there:
```
        pPanLabel->setText                  ( tr ( "PAN" ) );
        pcbMute->setText                    ( tr ( "MUTE" ) );
        pcbSolo->setText                    ( tr ( "SOLO" ) );
        strGroupBaseText =                    tr ( "GRP" );
```
If longer forms are fine, let me know and I update it to "BALANS".

Reply:
Thanks for the explanation. I think the short form is better here. I'm happy to approve it.